pkg/scheduler/scheduler.go

```diff
@@ -217,7 +217,7 @@ func (s *Scheduler) assign(ctx context.Context, e *entry) error {
 	go func() {
 		err := s.client.Update(ctx, newWorkload)
 		if err == nil {
-			s.recorder.Eventf(newWorkload, corev1.EventTypeNormal, "Assigned", fmt.Sprintf("Assigned to capacity %v", e.Capacity))
+			s.recorder.Eventf(newWorkload, corev1.EventTypeNormal, "Assigned", fmt.Sprintf("Assigned to capacity %v", newWorkload.Spec.AssignedCapacity))
```

Comment:
```suggestion
			s.recorder.Eventf(newWorkload, corev1.EventTypeNormal, "Assigned", "Assigned to capacity %v", newWorkload.Spec.AssignedCapacity)
```

Eventf already knows how to do formatting.